src/testers.jl

```diff
@@ -214,7 +218,7 @@ function test_rrule(
             fdm,
             (f, xs...) -> f(xs...; fkwargs...),
```

Comment:
should we do the same trick here, only insted of `call_on_copy` it is just `call`?
Which we would use here and on line 195?